components/browser/session/src/main/java/mozilla/components/browser/session/tab/CustomTabsService.kt

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.session.tab
+
+import android.app.Service
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.os.IBinder
+import android.support.customtabs.ICustomTabsCallback
+import android.support.customtabs.ICustomTabsService
+
+class CustomTabsService : Service() {
```

Comment:
I wonder if we should really put this into `session`. Some of the implementations here will require access to more things. How about moving this to feature-session? Or should we create something like feature-customtabs where we can add more things later on?

Reply:
Hm, not sure, let's file a follow-up and refactor this later?

Reply:
Okay, let's file a follow-up :) 👍